### ISSUE TITLE
Should check the libosmpbf in the CMake files

### ISSUE THREAD

**Issue by garaud** [Ifsttar/Tempus#51]
Hi,
When the `osm2tempus` was compiling, I got a:

```
~/tempus/src/loader/osm2tempus/osmpbfreader.h:39:34: fatal error: osmpbf/fileformat.pb.h: No such file or directory
```
I think this is the library `libosmpbf-dev` is missing. Is is possible to check if the library is installed in a `CMakeLists.txt` file?
Thanks,
Damien

**Comment by garaud** [comment]
And, of course, update the installation documentation.

**Comment by mhugo** [comment]
As discussed with @garaud, another option would be **not** to depend on osmpbf. After all, it is just a couple of files that are generated by protobuf from the osm (see here for instance https://github.com/substack/osm-pbf-parser/blob/master/lib/osmformat.proto ) and we could add these generated files to the compilation. The OSM PBF format won't change any soon

**Comment by mhugo** [comment]
Fixed in fabf0a668
